src/sentry/utils/sdk.py

```diff
@@ -99,6 +99,7 @@
     "sentry.tasks.app_store_connect.dsym_download": settings.SENTRY_APPCONNECT_APM_SAMPLING,
     "sentry.tasks.app_store_connect.refresh_all_builds": settings.SENTRY_APPCONNECT_APM_SAMPLING,
     "sentry.tasks.process_suspect_commits": settings.SENTRY_SUSPECT_COMMITS_APM_SAMPLING,
+    "sentry.tasks.process_commit_context": settings.SENTRY_SUSPECT_COMMITS_APM_SAMPLING,
```

Comment:
Once the code lands, let me know if you see the task actually being traced in production. I was not able to in when I added `sentry.tasks.reports.prepare_reports` to the list.